verify/verify.go

```diff
@@ -1158,6 +1202,16 @@ func RawTdxQuoteVerify(raw []byte, options *Options) error {
 	return TdxVerify(quote, options)
```

Comment:
You can change this to TdxAttestation() too

Reply:
Done